tests/credential/aembit/TestAccCredentialProviderResource.tf

```diff
@@ -1,11 +1,35 @@
 provider "aembit" {
 }
 
+
+resource "aembit_role" "role" {
+	name = "TF Acceptance Role for Token"
+	is_active = true
+	access_policies = { read = true, write = true }
+	client_workloads = { read = true, write = true }
+	trust_providers = { read = true, write = true }
+	access_conditions = { read = true, write = true }
+	integrations = { read = true, write = true }
+	credential_providers = { read = true, write = true }
+	server_workloads = { read = true, write = true }
+
+	agent_controllers = { read = true, write = true }
+
+	access_authorization_events = { read = true, write = true }

```

Comment:
events and logs only have read permission specified in the schema--do we have to specify write permissions here?